src/pages/Cart/ProductList.js

```diff
@@ -0,0 +1,117 @@
+import React, { useState } from 'react';
+import './ProductList.scss';
+import SIZE_LIST from './SizeList';
+
+function ProductList(props) {
+  const { product, onChangeAmount, cartDelete, setCheckItems, checkItems } =
+    props;
```

Comment:
```suggestion
function ProductList({ product, onChangeAmount, cartDelete, setCheckItems, checkItems }) {
```

구조분해할당 잘 해주셨습니다!

이 코드를 이런식으로 변경이 가능합니다!